.github/workflows/test.yml

```
@@ -0,0 +1,56 @@
+name: Test Postgres
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+    paths-ignore:
+      - "*.md"
+
+jobs:
+  build:
+    name: Test P${{ matrix.python-version }}, PSQL${{matrix.postgres-version}}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.7", "3.9"]
+        postgres-version: [10, 11, 12]
+        include:
+          - tox_test: "test_py37"
+            python-version: "3.7"
+          - tox_test: "test_py39"
+            python-version: "3.9"
+
+    services:
+      postgres:
+        # we need postgres with plpython extension postgres base docker hub image did not have that
+        # I use this one
+        image: thorinschiffer/postgres-plpython:${{ matrix.postgres-version }}
+        env:
+          POSTGRES_PASSWORD: postgres
+          # GreenplumPython use gpadmin as db name
+          POSTGRES_DB: gpadmin
```

"gpadmin" as a test database name is a little strange,
```suggestion
          POSTGRES_DB: greenplum_python_test
```